client/src/types/material-ui.d.ts

```diff
@@ -7,7 +7,7 @@ declare module '@mui/material/styles' {
     };
   }
 
-  interface ThemeOptions {
+  interface DeprecatedThemeOptions {
```

Comment:
@marvinsjsu , does this mean our theme doesn't work anymore?

Reply:
This change looks to be from the MUI5 package, but the themes should still work.  I've tested by updating some values on `client/src/theme/palette.ts` (for example changed text.primary to '#FF0000').